src/enhanced-text-input/tests/unit/EnhancedTextInput.ts

```diff
@@ -143,6 +143,7 @@ registerSuite('EnhancedTextInput', {
 						controls: 'foo',
 						describedBy: 'bar'
 					},
+					widgetId: 'textwidgetId',
```

Comment:
Should this be textWidgetId?

Reply:
oh, hah, I think this was a find and replace relic 😆